models/classes/ResultService.php

```diff
@@ -68,7 +68,8 @@ public function getDeliveryExecutionXml(DeliveryExecutionInterface $deliveryExec
      *
      * @param $deliveryId
      * @param $resultId
+     * @param bool $lastResults
      * @return mixed
      */
-    public function getQtiResultXml($deliveryId, $resultId);
+    public function getQtiResultXml($deliveryId, $resultId, $lastResults = false);
```

Comment:
Is this interface implemented anywhere else?

Reply:
I did not find any